sdk/src/main/java/io/radar/sdk/util/RadarLogBuffer.kt

```diff
@@ -15,15 +15,20 @@ internal interface RadarLogBuffer {
      * @param[level] log level
      * @param[message] log message
      */
-    fun write(level: Radar.RadarLogLevel, message: String, type: Radar.RadarLogType?, createdAt: Date = Date())
+    fun write(
+        level: Radar.RadarLogLevel,
+        type: Radar.RadarLogType?,
+        message: String,
+        createdAt: Date = Date()
+    )
 
     /**
      * Creates a stash of the logs currently in the buffer and returns them as a [Flushable] so that a successful
      * callback can cleanup this log buffer by deleting old log files.
      *
      * @return a [Flushable] containing all stored logs
      */
-    fun getFlushableLogsStash(): Flushable<RadarLog>
```

Comment:
convention is to have getter functions on iOS named `object` and on Android `getObject`. See  `RadarSettings.m` vs `RadarSettings.kt`.